docs/checks/checks.md

```diff
@@ -0,0 +1,166 @@
+
```

Comment:
Is this intended for another PR ? it is fine if this is just fixing some path issue

Reply:
This should be fixed with #996